proto/sf/substreams/rpc/v2/service.proto

```diff
@@ -130,6 +130,37 @@ message OutputDebugInfo {
 
 message ModulesProgress {
   repeated ModuleProgress modules = 1;
+  repeated Job running_jobs = 2;
+  repeated ModuleStats modules_stats = 3;
+}
+
+message Job {
+    uint64 stage = 1;
+    repeated string modules = 2;
+    uint64 start_block = 3;
+    uint64 stop_block = 4;
+    uint64 processed_blocks = 5;
+    uint64 duration_secs = 6;
+}
+
+message ModuleStats {
+    string name = 1;
+
+    // total_processed_blocks includes duplicate for each stage going over that module
+    uint64 total_processed_blocks = 2;
+    uint64 highest_processed_block = 3;
```

Comment:
Is this globally resolved? In the sense that it will always report the highest block of the "unified" view of the module and not just for the current run?

Reply:
I believe it could be globally resolved (from iniital store lookups), but lazily updated (if we are not at HEAD yet, and someone generates more files, we don't get notified)

Reply:
highest isn't so meaningful if there are plenty of segments that aren't finished, no? So perhaps highest contiguous? But then, it doesn't mean you can do anything at that highest block (maybe the first segment isn't finished, and everything on next stages are blocked on it)
Unsure of the use here.

Reply:
Ok, I will remove `highest_processed_block` because in the case of the mapper, we don't know and the definition is not obvious (if I'm asking for blocks 1000->2000 and the execout cache already has block 15M to 16M, what will 'highest_processed_block` show ?)

I will only keep the `highest_merged_block` value, for stores.
The `running_jobs` will allow further analysis of current state of output mapper jobs.